tasks/skylab/TagSortBam.wdl

```diff
@@ -3,13 +3,14 @@ version 1.0
 task CellSortBam {
   input {
     File bam_input
+    File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
-    Int machine_mem_mb = 100000
-    Int cpu = 2
-    Int disk = ceil(size(bam_input, "Gi") * 8)
-    Int preemptible = 3
+    String docker = "quay.io/kishorikonwar/secondary-analysis-python3-scientific:sctools-optimized8"
+    Int machine_mem_mb = 8000
+    Int cpu = 8
```

Comment:
should we default to 8 CPU? will that be a high compute VM?

Reply:
Maybe default to 4 so users don't have high bills and make a note that it is faster with 8 or 16 (then we can pass in 8 or 16 when we run it)

Reply:
reduced it to 4